### PR TITLE
Tidying lint warnings

### PR DESCRIPTION
**Pull request by domesticmouse**
Should we carry out this style of fixes, or am I holding it wrong?

### CODE REVIEW

File: examples/httpserver/bin/hello_world_server.dart [hunk 1]
```diff
@@ -18,9 +18,8 @@ Future main() async {
 
   // #docregion listen
   await for (HttpRequest request in server) {
-    request.response
-      ..write('Hello, world!')
-      ..close();
+    request.response.write('Hello, world!');
```
Comment (legalcodes) [inline examples/httpserver/bin/hello_world_server.dart]: Do these *_.server.dart examples need to be kept in sync with their counterparts in `dart-lang/dart-tutorials-samples`? Not a blocker for this PR, I'm mostly asking just to get a general sense of our approach to this right now.

See, e.g. https://github.com/dart-lang/dart-tutorials-samples/blob/master/httpserver/bin/hello_world_server.dart
(There are very slight differences between the code for these examples between dart-lang/site-www and dart-lang/dart-tutorials-samples)

File: examples/httpserver/bin/hello_world_server_secure.dart [hunk 2]
```diff
@@ -28,8 +28,7 @@ Future main() async {
   );
   print('Listening on localhost:${server.port}');
   await for (HttpRequest request in server) {
-    request.response
-      ..write('Hello, world!')
-      ..close();
+    request.response.write('Hello, world!');
+    await request.response.close();
```
Comment (legalcodes) [inline examples/httpserver/bin/hello_world_server_secure.dart]: Same question: see https://github.com/dart-lang/dart-tutorials-samples/blob/master/httpserver/bin/hello_world_server_secure.dart

File: examples/httpserver/bin/hello_world_server.dart (hunk 1 above)
Comment (domesticmouse) [reply examples/httpserver/bin/hello_world_server.dart]: I'll create a PR for dart-lang/dart-tutorials-samples repo to bring those samples up to date. I see anagram, for example, is stuck in Dart v1.

File: examples/httpserver/bin/hello_world_server_secure.dart (hunk 2 above)
Comment (domesticmouse) [reply examples/httpserver/bin/hello_world_server_secure.dart]: Aye, I'll do a full sweep of the tutorial samples

**Comment by domesticmouse** [comment]
I'll hold this PR open until Kathy has had a chance to review and sign off on it.

File: examples/httpserver/bin/hello_world_server.dart (hunk 1 above)
Comment (domesticmouse) [reply examples/httpserver/bin/hello_world_server.dart]: That took slightly longer than expected.

https://github.com/dart-lang/dart-tutorials-samples/pull/49

File: examples/httpserver/bin/hello_world_server_secure.dart (hunk 2 above)
Comment (domesticmouse) [reply examples/httpserver/bin/hello_world_server_secure.dart]: Also somewhere in https://github.com/dart-lang/dart-tutorials-samples/pull/49

**Comment by kwalrath** [comment]
I believe that #1717 will fix the build. Once that’s in, we should rerun the travis build, just to make sure…

**Pull request merged**